gobblin-yarn/src/main/java/gobblin/yarn/GobblinYarnConfigurationKeys.java

```diff
@@ -26,6 +26,9 @@
   public static final String APPLICATION_NAME_OPTION_NAME = "app_name";
   public static final String APP_QUEUE_KEY = GOBBLIN_YARN_PREFIX + "app.queue";
   public static final String APP_REPORT_INTERVAL_MINUTES_KEY = GOBBLIN_YARN_PREFIX + "app.report.interval.minutes";
+  public static final String MAX_GET_APP_REPORT_FAILURES_KEY = GOBBLIN_YARN_PREFIX + "max.get.app.report.failures";
+  public static final String EMAIL_NOTIFICATION_ON_SHUTDOWN_KEY =
```

Comment:
Actually the default is specified in `reference.conf` as required by Typesafe config.